blob/package.json

```diff
@@ -30,28 +34,26 @@
     "karma-mocha-reporter": "^2.2.5",
     "mocha": "^5.2.0",
     "rimraf": "^2.6.2",
-    "ts-loader": "^4.5.0",
-    "ts-node": "^7.0.1",
-    "tslint": "^5.11.0",
-    "typescript": "^3.0.1",
-    "uglifyjs-webpack-plugin": "^1.3.0",
-    "webpack": "^4.17.1",
-    "webpack-cli": "^3.1.0",
-    "zip-webpack-plugin": "^3.0.0"
+    "rollup": "^0.65.2",
+    "rollup-plugin-commonjs": "^9.1.8",
+    "rollup-plugin-multi-entry": "^2.0.2",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-replace": "^2.0.0",
+    "rollup-plugin-shim": "^1.0.0",
+    "rollup-plugin-uglify": "^5.0.2",
+    "typescript": "^3.0.1"
   },
   "scripts": {
     "test": "npm run test:node && npm run test:browser",
-    "test:node": "mocha --no-timeouts -r ts-node/register test/*.ts test/node/*.ts",
-    "test:browser": "npm run build:browsertest && karma start --single-run",
-    "build": "npm run build:node && npm run build:browser && npm run build:browsertest",
-    "build:node": "npm run build:cjs && npm run build:es6",
-    "build:cjs": "node ./node_modules/typescript/bin/tsc -p tsconfig.json",
-    "build:es6": "node ./node_modules/typescript/bin/tsc -p tsconfig.es6.json",
-    "build:browser": "webpack && rimraf browser/typings",
-    "build:browsertest": "webpack --config webpack.config.test.js && rimraf browser/test/typings",
+    "test:node": "npm run build:es6 && npm run build:nodebrowser && mocha --no-timeouts dist-esm/test/*.js dist-esm/test/node/*.js",
```

Comment:
`mocha --no-timeouts dist-esm/test/*.js dist-esm/test/node/*.js"` is not right, should be `mocha --no-timeouts dist-test/index.js`